web/beacon-app/src/components/MenuDropdown/MenuDropdown.tsx

```diff
@@ -8,16 +8,23 @@ interface RadixDropdownMenuProps {
   items: any;
   trigger?: React.ReactNode;
   isOpen?: boolean;
+  'data-cy'?: string;
   onOpenChange?: (isOpen: boolean) => void;
 }
 
-const MenuDropdownMenu = ({ items, trigger, isOpen, onOpenChange }: RadixDropdownMenuProps) => {
+const MenuDropdownMenu = ({
+  items,
+  trigger,
+  isOpen,
+  'data-cy': dataCy,
+  onOpenChange,
+}: RadixDropdownMenuProps) => {
   //console.log('items menu', items);
   return (
     <div className="relative">
       <DropdownMenuPrimitive.Root open={isOpen} onOpenChange={onOpenChange}>
-        <DropdownMenuPrimitive.Trigger>
-          <button className="border-none focus:ring-0">{trigger}</button>
+        <DropdownMenuPrimitive.Trigger className="border-none focus:ring-0" data-cy={dataCy}>
```

Comment:
`DropdownMenuPrimitive.Trigger` appears as a `button` in the DOM, so the other `button` was removed since we don't need a button inside of another button.